bodhi/tests/server/test_models.py

```diff
@@ -409,8 +409,9 @@ def test_same_build_types(self):
         # This should not raise any Exception.
         self.db.flush()
 
+    @mock.patch('bodhi.server.util.http_session')
     @mock.patch('bodhi.server.util.requests.get')
-    def test_get_pkg_committers_from_pagure(self, mock_get):
+    def test_get_pkg_committers_from_pagure(self, mock_get, session):
```

Comment:
Can we drop the mock on ```requests.get``` here? It seems like it isn't needed anymore, unless I am missing something which is possible :)

Reply:
I notice this mock is in place in the other bits of diff in this pull request, so same question on those.